docs/reference/elements/py-config.md

```diff
@@ -0,0 +1,242 @@
+      from data import make_x_and_y
+      x, y = make_x_and_y(n=1000)
+      fig, ax = plt.subplots()
+      ax.scatter(x, y)
+      fig
+    </py-script>
+  </body>
+</html>
+```
+
+The following optional values are supported by `<py-config>`:
+| Value | Type | Description |
+| ------ | ---- | ----------- |
+| `name` | string | Name of the user application. This field can be any string and is to be used by the application author for their own customization purposes. |
+| `description` | string | Description of the user application. This field can be any string and is to be used by the application author for their own customization purposes. |
+| `version` | string | Version of the user application. This field can be any string and is to be used by the application author for their own customization purposes. It is not related to the PyScript version. |
+| `schema_version` | number | The version of the config schema which determines what all keys are supported. This can be supplied by the user so PyScript knows what to expect in the config. If not supplied, the latest version for the schema is automatically used. |
+| `type` | string | Type of the project. The default is an "app" i.e. a user application |
+| `author_name` | string | Name of the author. |
+| `author_email` | string | Email of the author. |
+| `license` | string | License to be used for the user application. |
+| `autoclose_loader` | boolean | If false, PyScript will not close the loading splash screen when the startup operations finish. |
+| `packages` | List of Packages | Dependencies on 3rd party OSS packages are specified here. The default value is an empty list. |
+| `paths` | List of Paths | Local Python modules are to be specified here. The default value is an empty list. |
+| `plugins` | List of Plugins | List of Plugins are to be specified here. The default value is an empty list. |
+| `runtimes` | List of Runtimes | List of runtime configurations, described below. The default value contains a single Pyodide based runtime. |
+
+A runtime configuration consists of the following:
+| Value | Type | Description |
+| ----- | ---- | ----------- |
+| `src` | string (Required) | URL to the runtime source. |
+| `name` | string | Name of the runtime. This field can be any string and is to be used by the application author for their own customization purposes |
+| `lang` | string | Programming language supported by the runtime. This field can be used by the application author to provide clarification. It currently has no implications on how PyScript behaves. |
+
+Besides the above format, a user can also supply any extra keys and values that are relevant as metadata information or perhaps are being used within the application.
+
+For example, a valid config could also be with the snippet below:
+
+```
+<py-config type="toml">
+  magic = "unicorn"
+</py-config>
+```
+
+OR in JSON like
+
+```
+<py-config type="json">
+  {
+    "magic": "unicorn"
+  }
+</py-config>
+```
+
```

Comment:
I think it'd be extremely valuable to users if we provided an example of a full configuration with all the values available.

Reply:
Added complete schema in json + toml as examples